packages/mobile/src/app/saga.ts

```diff
@@ -156,6 +156,7 @@ export interface RemoteFeatureFlags {
   rewardsPercent: number
   rewardsStartDate: number
   rewardsMax: number
+  rewardsMin: number
```

Comment:
How have we been handling migrations/typing for these new remote config state variables? Technically, without a migration this type should be `undefined | number` right? In practice I understand that most users will connect to firebase on app load and have this populated.

Reply:
On the app reducer we have this:
```
case REHYDRATE: {
      // Ignore some persisted properties
      const rehydratePayload = getRehydratePayload(action, 'app')
      return {
        ...state,
        ...rehydratePayload,
        appState: initialState.appState,
        locked: rehydratePayload.requirePinOnAppOpen ?? initialState.locked,
        sessionId: '',
      }
    }
```
`state` in this case is the initial state, which has the default value for `rewardsMin` of `10` so I don't think this value will ever be `undefined` when read.